index.js

```diff
@@ -174,17 +174,17 @@
 //.
 //. There is a performance cost to run-time type checking. One may wish to
 //. disable type checking in certain contexts to avoid paying this cost.
-//. [`create`](#create) facilitates the creation of a Sanctuary module which
-//. does not perform type checking.
+//. Type checking is disabled by default if `process.env.NODE_ENV` is
+//. `'production'`.
 //.
-//. In Node, one could use an environment variable to determine whether to
-//. perform type checking:
+//. [`create`](#create) facilitates the creation of a Sanctuary module which
```

Comment:
Maybe it's just me (may be contextual) but I believe it implies you need to use `create` to enable type-checking and it is enabled by default on development and test environments. Correct?

Reply:
Good point. The sentence can be interpreted in two ways. I'll rework it to remove the ambiguity.

Reply:
I have updated this section of the documentation. Let me know what you think of the changes, @blackxored.